cg/models/mip/mip_config.py

```diff
@@ -13,29 +15,33 @@ class AnalysisType(BaseModel):
 class MipBaseConfig(BaseModel):
     """This model is used when validating the mip analysis config"""
 
-    family_id_: str = Field(None, alias="family_id")
-    case_id: str = None
-    analysis_type_: dict = Field(..., alias="analysis_type")
-    samples: list[AnalysisType] = None
-    config_path: str = Field(..., alias="config_file_analysis")
-    deliverables_file_path: str = Field(..., alias="store_file")
+    family_id_: str | None = Field(None, alias="family_id")
+    case_id: Annotated[str | None, Field(validate_default=True)] = None
```

Comment:
Probably out of scope here, but this seems odd. If family_id is only supported in older mip versions shouldn't we tie a specific mip config (i..e v1) to those versions? Should we address this in some way?

Reply:
No, it should be with a modern config.